src/Groups/gsets.jl

```diff
@@ -474,6 +485,33 @@ function stabilizer(Omega::GSet{T,S}, omega::S; check::Bool = true) where {T,S}
     return stabilizer(G, omega, gfun)
 end
 
+# support `stabilizer` under "derived" actions:
+# If the given point is a set, tuple, or vector of the element type of
+# the G-set then compute the setwise or pointwise stabilizer, respectively.
```

Comment:
Here similarly. The wait you explain it in the PR description is better understandablr